lobster/tools/codebeamer/codebeamer.py

```diff
@@ -206,32 +201,12 @@ def import_tagged(mh, cb_config, items_to_import):
     assert isinstance(mh, Message_Handler)
     assert isinstance(cb_config, dict)
     assert isinstance(items_to_import, set)
-    work_list = copy(items_to_import)
```

Comment:
please re-align the row below. There are many whitespace in there:
`rv        = []`